package.json

```diff
@@ -20,20 +20,39 @@
   "private": true,
   "devDependencies": {
     "@changesets/changelog-github": "^0.5.0",
+    "@changesets/cli": "^2.27.1",
+
     "@nx/js": "17.1.3",
     "@nx/plugin": "17.1.3",
-    "@types/node": "^20.11.30",
+    "@nx/eslint": "17.1.3",
+    "@nx/eslint-plugin": "17.1.3",
+    "nx": "17.1.3",
+
+    "@types/node": "^22.5.4",
+    "@types/react": "^18",
+
+    "@iarna/toml": "^2.2.5",
+    "eta": "^3.1.1",
+
     "eslint": "^8.53.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.4",
     "eslint-plugin-jsdoc": "^46.9.0",
     "eslint-plugin-simple-import-sort": "^10.0.0",
-    "eslint-plugin-storybook": "^0.6.15",
-    "nx": "17.1.3",
-    "tsx": "^4.7.1"
+
+    "tsx": "^4.7.1",
+    "typescript": "^5.2.2",
+    "resolve-tspaths": "^0.8.18",
+
+    "typedoc": "^0.26.7",
+    "typedoc-plugin-markdown": "^4.2.8",
+
+    "vitest": "^1.6.0"
   },
-  "dependencies": {
-    "@changesets/cli": "^2.27.1"
+  "peerDependencies": {
+    "viem": "^2.17.9",
+    "wagmi": "^2.12",
+
+    "react": "^18",
+    "@tanstack/react-query": "^5"
   }
```

Comment:
should these all be devDependencies? I dont think we want any peerDependencies at the root, since the root of this repo is not a package that will be installed by any consumers.